php/specimendao.class.inc

```diff
@@ -155,7 +155,6 @@ class SpecimenDAO extends \LORIS\Data\ProvisionerInstance
                  USING (SpecimenID)
                LEFT JOIN biobank_specimen_parent bspa
                  USING (SpecimenID)";
-        $params = [];
```

Comment:
the changes should be re-done. without them a bunch of notices would be generated